addons/point_of_sale/data/point_of_sale_data.xml

```diff
@@ -40,16 +40,6 @@
             <field name="is_pos_groupable">True</field>
         </record>
 
-        <record model="pos.bill" id="0_01" forcecreate="0">
```

Comment:
Override in `odoo/addons/l10n_in_pos/data/pos_bill_data.xml` needs to be removed.

```
        <record model="pos.bill" id="point_of_sale.0_01">
            <field name="for_all_config">False</field>
        </record>

        <record model="pos.bill" id="point_of_sale.0_02">
            <field name="for_all_config">False</field>
        </record>
```

Reply:
@davidmonnom changes are done please check.
Thanks!